### ISSUE TITLE
Error installing npm zlib package

### ISSUE THREAD

**Issue by rtorr** [vercel/turborepo#270]
### What version of Turborepo are you using?

1.0.9

### Describe the Bug

Error when running `npm install turbo -D`.

```
npm ERR! path C:\Users\<username>\work\<project-name>\node_modules\zlib
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c node-waf clean || true; node-waf configure build
npm ERR! 'node-waf' is not recognized as an internal or external command,
npm ERR! operable program or batch file.
npm ERR! 'true' is not recognized as an internal or external command,
npm ERR! operable program or batch file.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\<username>\AppData\Local\npm-cache\_logs\2021-12-14T17_10_16_275Z-debug.log
```

I can reproduce this by just trying to install `zlib@1.0.5` directly via `npm install zlib@1.0.5`.

zlib is now included in newer versions of Nodejs, but im guessing you are using it because of browserify?

### Expected Behavior

No error.

### To Reproduce

On windows (also seems to be the case using wsl) using `node@16.10.0` and `npm@7.24.0` try to install `zlib@1.0.5`.

**Comment by rtorr** [comment]
Oddly, things seem to install fine with yarn 🤔 ?

**Comment by ez-alexfrazer** [comment]
seeing the same here as well

**Comment by jaredpalmer** [comment]
Fixed with #308
